cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -381,6 +381,7 @@ CAST_NUMERIC_FROM_STRING(double, arrow::DoubleType, FLOAT8)
 
 GDV_FN_CAST_VARCHAR_INTEGER(int32, Int32Type)
 GDV_FN_CAST_VARCHAR_INTEGER(int64, Int64Type)
+GDV_FN_CAST_VARCHAR_INTEGER(date64, Date64Type)
```

Comment:
maybe rename the macro to GDV_FN_CAST_VARCHAR and remove GDV_FN_CAST_VARCHAR_REAL since it seems to be doing the same thing,

Reply:
Ok ignore this. Looks like the formatter is different for both the types

Reply:
The suggested changes were applied

Reply:
Fixed